### DISCUSSION TITLE
Paraview Installed but Conda Cannot Find It

### DISCUSSION THREAD

**Discussion by ParadoxPrism117** [idaholab/moose#28628] [Q&A Tools] [answered]
I have been trying to visualize the results from a simulation I ran using Paraview but I cannot get the software to run. I followed the steps on conda-forge. https://github.com/conda-forge/paraview-feedstock

However, when I went to install it, it replied "# All requested packages already installed". I checked "conda list" and it was listed in the environment but when I tried to activate it using "conda activate paraview" it could not find the conda environment.

It sounds like it IS installed, but it cannot find it?

**Comment by GiudGiud** [comment]
hello

what does `which paraview` return in the command line?
Did you create a separate environment for paraview? Or did you simply install it in your current one?

**Reply by ParadoxPrism117** [reply]
it returns 

/home/user/miniforge/bin/paraview

It shouldn't be in a special environment, or at least I didn't intend it to be in one.

**Reply by GiudGiud** [reply]
can you just type `paraview` in the console to run it then?

**Reply by ParadoxPrism117** [reply]
Response: "Segmentation fault (core dumped)"

**Reply by GiudGiud** [reply, marked as answer]
So it is installed it just does not run.
The conda install of paraview is likely going to be difficult to debug.
I recommand you re install from the online binaries
https://www.paraview.org/download/

**Reply by milljm** [reply]
This usually indicates you're missing core OS dependencies (like mesa, nvidia, etc).

What happens if you instruct your system to install paraview using it's package manager (like `apt-get install paraview`)? If thats a possibility, this will usually bring in those necessary dependencies that Conda can't satisfy. You don't have to install paraview in this way if you don't want to (answer "no"). Just by virtue of the install attempt, it'll list whats missing and give an idea whats required.

**Reply by ParadoxPrism117** [reply]
> This usually indicates you're missing core OS dependencies (like mesa, nvidia, etc).
> 
> What happens if you instruct your system to install paraview using it's package manager (like `apt-get install paraview`)? If thats a possibility, this will usually bring in those necessary dependencies that Conda can't satisfy. You don't have to install paraview in this way if you don't want to (answer "no"). Just by virtue of the install attempt, it'll list whats missing and give an idea whats required.

Response to "apt-get install paraview":

E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?

**Reply by ParadoxPrism117** [reply]
> So it is installed it just does not run. The conda install of paraview is likely going to be difficult to debug. I recommand you re install from the online binaries https://www.paraview.org/download/

I downloaded the tar.gz folder but I don't see any kind of configure or make file to actually install paraview

**Reply by GiudGiud** [reply]
should just be some clicking with those files. no make or configure

**Reply by milljm** [reply]
So did the download package work from www.paraview.org? That is odd in my opinion, as Conda installs the very same "binary only, no need to configure or compile".

I would assume the same errors would occur due to missing system dependencies with the downloaded tarball from www.paraview.org (the tarball would not install mesa, nvidia, etc).

But I see you selected that as the appropriate solution.

Edit:
Also, the error you received with `apt-get` was because you need to use `sudo` in order to run it. But if everything is working, no need to run this command anymore!

**Reply by jackeggemeyer** [reply]
I am running into the same issue, if you go into the paraview folder you can launch it with no issues. Just hit the thing with the same logo as the -opt files. How can I force conda to be aware of the install so I can use the terminal?

**Reply by GiudGiud** [reply]
if you know where paraview is installed then you can add that folder to the PATH environment variable.
Alternatively you can set an alias that points to this executable
